Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
```

Comment:
Maybe we can be clearer about the distinction here? Maybe "Programming Fundamentals" and "The Javascript Language". Those of us really familiar with all of this can infer the distinction easily enough, but that might not be so for everyone (perhaps many people?) reading this doc.